### PR TITLE
Create data models

### PR DESCRIPTION
**Pull request by alanluqman**
## Description
In this PR, I created data models

### CODE REVIEW

File: db/migrate/20221020182851_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (ShahierNashaat) [inline db/migrate/20221020182851_create_comments.rb]: - [x] Kindly write the query like this so the foreign key name will be `author_id`.

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

File: db/migrate/20221020182834_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :post, null: false, foreign_key: true
```
Comment (ShahierNashaat) [inline db/migrate/20221020182834_create_likes.rb:4]: - [x] Kindly also here write the query like this so the foreign key name will be `author_id`.

```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @alanluqman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alanluqman** [comment]
Thanks for your review, dear @ShahierNashaat I fixed all

**Review by BenMukebo — APPROVED** [review]
### Hi @alanluqman, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Database adapter is using Postgresql 👍🏻 
- All the migration file have been creating well 🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alanluqman** [comment]
Thanks for your review dear.

**Pull request merged**